WordPress/Classes/ViewRelated/NUX/WPStyleGuide+Login.swift

```diff
@@ -83,6 +85,12 @@ extension WPStyleGuide {
         return UIFont.systemFont(ofSize: fontToGetSize.pointSize, weight: .medium)
     }
 
+    class func mediumWeightFont(forStyle style: UIFontTextStyle, maximumPointSize: CGFloat) -> UIFont {
```

Comment:
This is only being called in one place, with the size argument being `maxFontSizeForLogin`. Might as well put that into the `Constants` and use it directly instead of taking it as an argument.

Reply:
I thought about it as well. But there are still a few other views in the login flow where I need to limit the maximum font size as well, and I thought it'd be better to wait until the whole login flow is done to see if passing that maximum font size as a parameter was necessary. I'll remove it for now, I can always bring it back in later if necessary ;)

Reply:
On second thought... I think it would be better to pass the maximum font size as a parameter. `mediumWeigthFont` is also used to style cells, and standalone labels in other parts of the login flow, passing different styles (i.e. `LoginSocialErrorCell` or `SignupUsernameTableViewController`). Do we want to make the maximum font size the same in those different contexts?